cvat/apps/documentation/installation.md

```diff
@@ -0,0 +1,308 @@
+    instructions can be found [here](https://docs.docker.com/docker-for-windows/install/). Note:
+    Docker Desktop requires Windows 10 Pro or Enterprise version 14393 to run.
+
+-   Download and install
+    [Git for Windows](https://github.com/git-for-windows/git/releases/download/v2.21.0.windows.1/Git-2.21.0-64-bit.exe).
+    When installing the package please keep all options by default.
+    More information about the package can be found [here](https://gitforwindows.org).
+
+-   Download and install [Google Chrome](https://www.google.com/chrome/). It is the only browser
+    which is supported by CVAT.
+
+-   Go to windows menu, find `Git Bash` application and run it. You should see a terminal window.
+
+-   Clone _CVAT_ source code from public
+    [GitHub repository](https://github.com/opencv/cvat).
+
+    ```bash
+    git clone https://github.com/opencv/cvat
+    cd cvat
+    ```
+
+-   Build docker images by default. It will take some time to download public
+    docker image ubuntu:16.04 and install all necessary ubuntu packages to run
+    CVAT server.
+
+    ```bash
+    docker-compose build
+    ```
+
+-   Run docker containers. It will take some time to download public docker
+    images like postgres:10.3-alpine, redis:4.0.5-alpine and create containers.
+
+    ```sh
+    docker-compose up -d
+    ```
+
+-   You can register a user but by default it will not have rights even to view
+    list of tasks. Thus you should create a superuser. The superuser can use
+    admin panel to assign correct groups to the user. Please use the command
+    below:
+
+    ```sh
+    winpty docker exec -it cvat bash -ic 'python3 ~/manage.py createsuperuser'
+    ```
+    Choose login and password for your admin account. For more information
+    please read [Django documentation](https://docs.djangoproject.com/en/2.2/ref/django-admin/#createsuperuser).
+
+-   Open the installed Google Chrome browser and go to [localhost:8080](http://localhost:8080).
+    Type your login/password for the superuser on the login page and press _Login_
+    button. Now you should be able to create a new annotation task. Please read
+    documentation about CVAT for more details.
+
+## Mac OS Mojave
+-   Download [Docker for Mac](https://download.docker.com/mac/stable/Docker.dmg).
+    Double-click Docker.dmg to open the installer, then drag Moby the whale
+    to the Applications folder. Double-click Docker.app in the Applications
+    folder to start Docker. Complete instructions can be found
+    [here](https://docs.docker.com/v17.12/docker-for-mac/install/#install-and-run-docker-for-mac).
+
+-   There are several ways to install Git on a Mac. The easiest is probably to
+    install the Xcode Command Line Tools. On Mavericks (10.9) or above you can
+    do this simply by trying to run git from the Terminal the very first time.
+
+    ```bash
+    git --version
+    ```
+
+    If you don’t have it installed already, it will prompt you to install it.
```

Comment:
If you haven't installed git yet, the terminal will prompt ..

Reply:
I got the sentence from git's install manual. I believe it should be OK from gramma point of view.